xapian-applications/omega/Makefile.am

```diff
@@ -95,11 +95,11 @@ TESTS =	atomparsetest$(EXEEXT)\
 	jsonesctest$(EXEEXT)\
 	md5test$(EXEEXT)\
 	omegatest\
+	omindextest\
```

Comment:
This is a compiled test program, so ought to have the `$(EXEEXT)` suffix here (for platforms which require `.exe` on compiled executables).

Reply:
Oh sorry, I was mixing up `omindexcheck` and `omindextest` - this is correct as-is.